### PR TITLE
Remove dependency on vast::path in pcap formatter

### PR DESCRIPTION
**Pull request by JoeLoser**
###  :notebook_with_decorative_cover: Description

Problem:
- `pcap` formatter depends on `vast::path` for checking if the input
  exists.

Solution:
- Use `std::filesystem::exists` instead.

Note:
- Did not change `input_` data member to be a `std::filesystem::path`
  type since the pcap formatter allows for the input to be `-` meaning
  `stdin` which isn't well representable using `std::filesystem::path`.

###  :memo: Checklist

- [ ] All user-facing changes have changelog entries.
- [ ] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [ ] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

File-by-file.

### CODE REVIEW

**Review by mavam — APPROVED** [review]
Looks good to me.

File: libvast/src/format/pcap.cpp [hunk 1]
```diff
@@ -27,7 +26,7 @@
 #  include <caf/settings.hpp>
 
 #  include <cstddef>
-#  include <pcap.h>
```
Comment (JoeLoser) [inline libvast/src/format/pcap.cpp:30]: @dominiklohmann looks like a rebase error in your top-commit. I'll rebase and push to this branch in a minute.

Comment (dominiklohmann) [reply libvast/src/format/pcap.cpp:30]: Yes sorry, that shouldn't have been removed. Thanks for the quick response. We were just looking at which PRs to merge in what order now that they're unblocked.

We gave you a shoutout in the release notes and the release blog post btw. 😊

Comment (JoeLoser) [reply libvast/src/format/pcap.cpp:30]: No problem, this PR should be good now.

Thanks for the kind words in the release notes and blog post -- just gave them a read 😊

**Pull request merged**